### PR TITLE
Update water.yaml

### PR DESCRIPTION
**Pull request by VassilisDaioglou**
Added water withdrawal/consumption for electricity production.

### CODE REVIEW

File: definitions/variable/water/water.yaml [hunk 1]
```diff
@@ -12,6 +12,10 @@
     definition: Water withdrawal for irrigation
     unit: km3/yr
     tier: 2
+- Water Withdrawal|Electricity:
+    definition: Total water withdrawals for net electricity production
+    unit: km3/yr
+    tier: 2
```
Comment (danielhuppmann) [inline definitions/variable/water/water.yaml:18]: I guess we want 
```suggestion
- Water Withdrawal|Electricity:
    definition: Total water withdrawals for total electricity generation
    unit: km3/yr
    tier: 2
- Water Withdrawal|Electricity|{Electricity Source}:
    definition: Total water withdrawals for electricity generation from {Electricity Source}
    unit: km3/yr
    tier: 2
```

File: definitions/variable/water/water.yaml [hunk 2]
```diff
@@ -31,6 +35,10 @@
     definition: Water consumption for irrigation
     unit: km3/yr
     tier: 2
+- Water Consumption|Electricity:
+    definition: Total water withdrawals for net electricity production
+    unit: km3/yr
+    tier: 2
```
Comment (danielhuppmann) [inline definitions/variable/water/water.yaml:41]:
```suggestion
- Water Consumption|Electricity:
    definition: Total water withdrawals for total electricity generation
    unit: km3/yr
    tier: 2
- Water Consumption|Electricity|{Electricity Source}:
    definition: Total water withdrawals for electricity generation from {Electricity Source}
    unit: km3/yr
    tier: 2
```

**Review by danielhuppmann — COMMENTED** [review]
I guess we also want to have water-use for electricity generation by source?

File: definitions/variable/water/water.yaml (hunk 2 above)
Comment (VassilisDaioglou) [reply definitions/variable/water/water.yaml:41]: This would require adding a "tag_electricity-source.yaml", right?

Comment (danielhuppmann) [reply definitions/variable/water/water.yaml:41]: That's the beauty of the tag-lists...
This particular one already exists and this way, we can make sure that Secondary Energy|Electricity|..., Water Consumption|Electricity|..., technical parameters... all use the same elecitricity-generation technologies. https://github.com/IAMconsortium/common-definitions/blob/main/definitions/variable/energy/tag_secondary_electricity_sources.yaml

Comment (danielhuppmann) [reply definitions/variable/water/water.yaml:41]: For context, the location of any files within the "variable" folder does not have any functional impact, sub-folders are just to help us humans structure the info. For the code, it's all the same, so you can reuse across sub-folders.

File: definitions/variable/water/water.yaml [hunk 3]
```diff
@@ -12,6 +12,18 @@
...
+- Water Withdrawal|Electricity|{Electricity Source}:
+    definition: Total water withdrawals for electricity generation from {Electricity Source}
+    unit: km3/yr
+    tier: 2
+- Water Withdrawal|Electricity|{Electricity Source}:
+    definition: Total water withdrawals for electricity generation from {Electricity Source}
+    unit: km3/yr
+    tier: 2
```
[the remark this answers is not in the document]
Comment (VassilisDaioglou) [reply definitions/variable/water/water.yaml:26]: Yep..sorry

**Review by danielhuppmann — APPROVED** [review]
Looks good, thanks!

**Pull request merged**